cli/src/commands/create-expo-stack.ts

```diff
@@ -316,15 +309,23 @@ const command: GluegunCommand = {
         const generateRerunScript = (cliResults: CliResults) => {
           let script = `npx create-expo-stack ${cliResults.projectName} `;
 
-          if (cliResults.packages.length > 0 && cliResults.packages[0].name === 'nativewindui') {
+          const nativewindUISelected = cliResults.packages.find((p) => p.name === 'nativewindui');
```

Comment:
This doesn't seem like a very good naming convention given how this value is now used to both assert that the user has selected NativeWindUI as a styling option AND check which components they have selected.

What are your thoughts on the following approach instead?
```
// Check if NativeWindUI is selected as the styling package
const isNativeWindUISelected = cliResults.packages.some((p) => p.name === 'nativewindui');

// Get the selected NativeWindUI package details (including components)
const nativeWindUIComponents = cliResults.packages.find((p) => p.name === 'nativewindui')?.options?.selectedComponents || [];
```

Reply:
sounds better to me 👍